app/cafe.py

```diff
@@ -0,0 +1,17 @@
+from app.errors import (NotVaccinatedError, NotWearingMaskError,
+                        OutdatedVaccineError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")

```

Comment:
The error message for `NotVaccinatedError` should be more descriptive. Consider specifying that the visitor is not vaccinated, rather than using a generic message.